src/features/game/expansion/lib/boosts.ts

```diff
@@ -100,7 +100,7 @@ export const getFoodExpBoost = (
   }
 
   //Bumpkin Skill Boost Curer
-  if (food.building === "Deli" && skills["Curer"]) {
+  if ("building" in food && food.building === "Deli" && skills["Curer"]) {
```

Comment:
Is this extra check needed? ("building" in food)

Otherwise `food.building === "Deli"` would be `undefined === 'Deli` which would still be false

Reply:
@adamhannigan It was the typescript complaining, as not all consumables have a building.

I added a type guard here instead of inspecting the object keys.